src/styles/cards.scss

```diff
@@ -0,0 +1,99 @@
+.cards {
+  row-gap: 46px;
+  column-gap: 48px;
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  padding: 50px 40px 72px;
+
```

Comment:
```suggestion
  grid-template-columns: repeat(4, 200px);
  padding: 50px 40px 72px;
  justify-content: center;
```
grid doesn't match the design.
my suggestions should help to fix it

![Screenshot from 2023-09-25 19-13-22](https://github.com/yarkob/layout_catalog/assets/33029864/2dd66b00-866b-445c-8127-c0b2c91c205d)